path_config_test.go

```diff
@@ -44,46 +44,92 @@ func setupLocalFiles(t *testing.T, b logical.Backend) func() {
 }
 
 func TestConfig_Read(t *testing.T) {
-	b, storage := getBackend(t)
-
-	cleanup := setupLocalFiles(t, b)
-	defer cleanup()
-
-	data := map[string]interface{}{
-		"pem_keys":               []string{testRSACert, testECCert},
-		"kubernetes_host":        "host",
-		"kubernetes_ca_cert":     testCACert,
-		"issuer":                 "",
-		"disable_iss_validation": false,
-		"disable_local_ca_jwt":   false,
-	}
-
-	req := &logical.Request{
-		Operation: logical.UpdateOperation,
-		Path:      configPath,
-		Storage:   storage,
-		Data:      data,
-	}
-
-	resp, err := b.HandleRequest(context.Background(), req)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%s resp:%#v\n", err, resp)
+	type args struct {
```

Comment:
I think we can probably move `data` to the tests structure, and then drop `args`.